Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.22-alpine AS builder
+
+WORKDIR /app
+
+COPY go.mod .
+COPY go.sum .
+COPY ./src ./src
+
+RUN go mod download
+RUN go mod verify
+
+RUN go build -ldflags="-w -s" -o server ./src/server
```

Comment:
so again src/ is a java thing not a go thing

Reply:
Noted. Thanks!

Reply:
I used `src` to organize files that I want to copy during the Docker build stage so that I wouldn't have to be writing separate lines to add files to Docker build. What do you think @ldemailly?